<Issue id="15852" key="FABN-137" number="137" project="10604" reporter="rameshthoomu" assignee="jimthematrix" creator="rameshthoomu" type="10004" summary="e2e tests are failing on sdk-node commit# 9d70798" priority="1" resolution="10000" status="6" created="2017-04-19 19:01:46.0" updated="2018-07-19 00:52:59.0" resolutiondate="2017-04-20 17:59:16.0" votes="0" watches="1" workflowId="33868"> <description><! CDATA Observed sdk-node e2e tests are failing with the below error on commit# 9d70798  Below is the error log:  {code:java} Attaching to ca_peerOrg2, orderer0, ca_peerOrg1, couchdb, peer0, peer2, peer1, peer3  36mca_peerOrg2 | 0m 2017/04/19 18:51:45  INFO  Configuration file location: /etc/hyperledger/fabric-ca-server/fabric-ca-server-config.yaml  36mca_peerOrg2 | 0m Error: ca.name property is required but is missing from the configuration file  36mca_peerOrg2 | 0m Usage:  {code} Docker Log File:  https://jenkins.hyperledger.org/sandbox/job/e2e_FIT_x86_64/6/artifact/gopath/src/github.com/hyperledger/fabric-sdk-node/test/fixtures/dockerlogfile.log =======> FABRIC PEER COMMIT NUMBER - b6201fe =======> FABRIC CA COMMIT NUMBER - e583181 =======> FABRIC SDK NODE COMMIT NUMBER - 9d70798  ></description> </Issue>
